Source/Icebreaker/Cards/PairUpNotificationCard.cs

```diff
@@ -22,30 +22,30 @@ public class PairUpNotificationCard
         /// <param name="teamName">The team name.</param>
-        /// <param name="firstPerson">The first person in the pair up.</param>
-        /// <param name="secondPerson">The second person in the pair up.</param>
+        /// <param name="sender">The first person in the pair up.</param>
+        /// <param name="recipient">The second person in the pair up.</param>
         /// <param name="botDisplayName">The bot display name.</param>
         public static Attachment GetCard(
             string teamName,
-            ChannelAccount firstPerson,
-            ChannelAccount secondPerson,
+            ChannelAccount sender,
+            ChannelAccount recipient,
             string botDisplayName)
         {
-            var person1 = firstPerson.Properties.ToObject<TeamsChannelAccount>();
-            var person2 = secondPerson.Properties.ToObject<TeamsChannelAccount>();
-
-            var firstPersonFirstName = string.IsNullOrEmpty(person1.GivenName) ? firstPerson.Name : person1.GivenName;
-            var secondPersonFirstName = string.IsNullOrEmpty(person2.GivenName) ? secondPerson.Name : person2.GivenName;
-            var title = string.Format(Resources.MeetupTitle, firstPersonFirstName, secondPersonFirstName);
+            var senderGivenName = string.IsNullOrEmpty(sender.Properties.ToObject<TeamsChannelAccount>().GivenName) ?
+                sender.Name : sender.Properties.ToObject<TeamsChannelAccount>().GivenName;
+            var recipientGivenName = string.IsNullOrEmpty(recipient.Properties.ToObject<TeamsChannelAccount>().GivenName) ?
+                recipient.Name : recipient.Properties.ToObject<TeamsChannelAccount>().GivenName;
+            var title = string.Format(Resources.MeetupTitle, senderGivenName, recipientGivenName);
 
             var escapedTitle = Uri.EscapeDataString(title);
             var content = string.Format(Resources.MeetupContent, botDisplayName);
             var escapedContent = Uri.EscapeDataString(content);
 
-            var isGuest = IsGuestUser(person2);
-            var personUpn = IsGuestUser(person2) ? person2.Email : person2.UserPrincipalName;
-            var meetingLink = "https://teams.microsoft.com/l/meeting/new?subject=" + escapedTitle + "&attendees=" + personUpn + "&content=" + escapedContent;
-            var chatMessageLink = $"https://teams.microsoft.com/l/chat/0/0?users={personUpn}&message=Hi%20there%20";
+            var recipientUpn = !IsGuestUser(recipient.Properties.ToObject<TeamsChannelAccount>()) ?
+                recipient.Properties.ToObject<TeamsChannelAccount>().UserPrincipalName :
+                recipient.Properties.ToObject<TeamsChannelAccount>().Email;
+            var meetingLink = $"https://teams.microsoft.com/l/meeting/new?subject=" + escapedTitle + "&attendees=" + recipientUpn + "&content=" + escapedContent;
+            var chatMessageLink = $"https://teams.microsoft.com/l/chat/0/0?users={recipientUpn}&message=Hi%20there%20";
```

Comment:
@coderkrishna "Hi there" should come from a resource.

Reply:
Handled